cime_config/machines/config_workflow.xml

```diff
@@ -37,28 +37,28 @@
       <template>template.case.test</template>
       <prereq>$BUILD_COMPLETE and $TEST</prereq>
     </job>
-    <!--job name="case.post_run_io">
+    <job name="case.post_run_io">
       <template>template.post_run_io</template>
       <dependency>case.run</dependency>
-      <prereq>case.get_value("PIO_TYPENAME_ATM") == 'adios' or \
-              case.get_value("PIO_TYPENAME_CPL") == 'adios' or \
-              case.get_value("PIO_TYPENAME_OCN") == 'adios' or \
-              case.get_value("PIO_TYPENAME_WAV") == 'adios' or \
-              case.get_value("PIO_TYPENAME_GLC") == 'adios' or \
-              case.get_value("PIO_TYPENAME_ICE") == 'adios' or \
-              case.get_value("PIO_TYPENAME_ROF") == 'adios' or \
-              case.get_value("PIO_TYPENAME_LND") == 'adios' or \
-              case.get_value("PIO_TYPENAME_ESP") == 'adios' or \
-              case.get_value("PIO_TYPENAME_IAC") == 'adios'</prereq>
+      <prereq>case.get_value("PIO_TYPENAME_ATM").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_CPL").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_OCN").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_WAV").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_GLC").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_ICE").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_ROF").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_LND").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_ESP").startswith('adios') or \
+              case.get_value("PIO_TYPENAME_IAC").startswith('adios')</prereq>
       <runtime_parameters>
         <walltime>0:30:00</walltime>
       </runtime_parameters>
-    </job-->
+    </job>
     <job name="case.st_archive">
```

Comment:
Does this mean the conversion is run as part of the short-term archive job?

Reply:
The conversion job is only run if user sets the environment variable SPIO_ENABLE_ADIOSBP2NC_CONVERSION to 1 (by default it's not set). If the env variable is set the conversion job is run before the archive job

Reply:
(The conversion job would be a separate batch job - not part of the archive job)